chapter01/c01_downstairs_badend.rpy

```diff
@@ -17,33 +17,33 @@ label c01_downstairs_badend:
     Stealthfully, I walked towards the elevator. [Pause] It would be a lot more troublesome to take the stairs.
     The other doctors and nurses would have a higher chance of noticing me if I took the stairs, [Pause] and they would instantly notice me thanks to the incident four years ago.
     Once I reached the elevator, I quickly pressed the button five times in desperation while looking both to the left and right, making sure not to get myself caught..
-    I took a step back and patiently waited for the elevator. I heard two low voices- most likely two men- shouting. [Pause] Their voices were mixed with the rapid sounds of stomping footsteps.
+    I took a step back and patiently waited for the elevator. I heard two low voices - most likely two men - shouting. [Pause] Their voices were mixed with the rapid sounds of stomping footsteps.
     I gasped softly in worry, trying to find a place to hide.
-    Man's Voice "Did... see... poor kid?!"
-    I froze in place at the one of the voice's words. [Pause] A child... Could this be about the noise downstairs?
+    mans_voice "Did... see... poor kid?!"
```

Comment:
For these to (eventually) work in ren'py, I've changed them to lowercase variables